docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -970,6 +970,27 @@ How to **investigate**:
        $.apps.v1.statefulSet.spec.template.metadata.withLabelsMixin({ [$._config.gossip_member_label]: 'false' }),
      ```
 
+### MimirKubernetesServiceEndpointsApproachingLimit
+
+This alert fires if a Kubernetes service used by Mimir for service discovery is approaching the endpoints limit.
+
+See [MimirKubernetesServiceEndpointsOverLimit](#MimirKubernetesServiceEndpointsOverLimit).
+
+### MimirKubernetesServiceEndpointsOverLimit
+
+This alert fires if a Kubernetes service used by Mimir for service discovery has reached the endpoints limit.
+
+How it **works**:
+
+- By default, the Kubernetes control plane creates and manages service endpoints to have no more than 100 endpoints each ([doc](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/)). You can configure this limit with the `--max-endpoints-per-slice kube-controller-manager` flag, up to a maximum of 1000.
+- Once a Kubernetes service endpoints list reached the limit, the endpoints are truncated to the limit.
+- Mimir uses some Kubernetes services for DNS-based service discovery. If the endpoints of one of such services is getting truncated, Mimir is unable to discover all pods.
+
+How to **fix** it:
+
+- Increase Kubernetes `--max-endpoints-per-slice kube-controller-manager` to 1000, if it's currently running with a lower value. Configure the limit in the alert to reflect the new setting.
```

Comment:
```suggestion
- Increase the Kubernetes `--max-endpoints-per-slice kube-controller-manager` limit to 1000, if it's currently running with a lower value. Configure the limit in the alert to reflect the new setting.
```

Reply:
If `--max-endpoints-per-slice kube-controller-manager` is a limit, it's good to specify that here. Same if it's a flag, setting, etc.